autoconnect/autoconnect-common/src/test_support.rs

```diff
@@ -15,20 +15,37 @@ pub const HELLO: &str = r#"{"messageType": "hello", "use_webpush": true}"#;
 pub const HELLO_AGAIN: &str = r#"{"messageType": "hello", "use_webpush": true,
                                   "uaid": "deadbeef-0000-0000-deca-fbad00000000"}"#;
 
+pub const CURRENT_MONTH: &str = "message_2023_5";
```

Comment:
I'm tempted to suggest that we either use the current, production value here (`message_2017_09` IIRC), or remove the table rotation code entirely since it's no longer useful and just adds complications.

Reply:
Changed this test value to match prod, of course we can't hardcode the result of `Db::message_month` to such a value though (in case you're suggesting that which you probably aren't).

Reply:
With the metrics we saw yesterday I'll definitely remove the table migration code in a separate PR (mostly to document it in isolation). I think it will end up having us match autoendpoint's `validate_webpush_user` even more closely.